nemo/collections/nlp/data/dialogue/dataset/dialogue_bert_dataset.py

```diff
@@ -82,15 +87,52 @@ def __init__(self, dataset_split: str, dialogues_processor: object, tokenizer, c
             ignore_extra_tokens=self.cfg.ignore_extra_tokens,
             ignore_start_end=self.cfg.ignore_start_end,
         )
-
+        
         self.all_input_ids = features[0]
         self.all_segment_ids = features[1]
         self.all_input_mask = features[2]
         self.all_loss_mask = features[3]
         self.all_subtokens_mask = features[4]
         self.all_slots = features[5]
+
+        # drive through dataset: label_of_other_type=0; assistant dataset: label_of_other_type=54
+        # use the train_slot_stats.tsv file majority class as the "Other" slot class
+        file_path_for_stats = cfg.data_dir+"/train_slot_stats.tsv"
+        
+        with open(file_path_for_stats) as f:
+            label_of_other_type = int(f.read().strip().split('\n')[0].split()[0])
+
+        bio_train_slots = [DialogueBERTDataset.get_bio_slot_label_from_sequence(t, label_of_other_type) for t in features[5]]
+        self.all_bio_slots = bio_train_slots
+
+        bio_mention_labels = [DialogueBERTDataset.get_bio_mention_labels(t, bio_list) for t, bio_list in zip(features[5], bio_train_slots)]
```

Comment:
why not use self.all_slots instead of features[5] since you have already assigned it in line 96 and it will make it much more readable?